resources/views/movies/create.blade.php

```diff
@@ -0,0 +1,58 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+            {{ __('Create a new film entity') }}
+        </h2>
+    </x-slot>
+    <x-auth-card>
```

Comment:
I would probably try to create something similar to your wireframe, instead of having the authentication card component here. 

The `x-slot` header will not show and the reason is because of the changes on the x-app-layout.

Reply:
Just using the auth-card as a temporary layout. I plan on using something similar to it for the movie create and edit

Reply:
Got it ;D
I would suggest you start looking at this now ;D otherwise you would have to change all the places you will use it, and sometimes that is hard to keep track :D

Reply:
Update design has been implemented. Might need some further improvements however add and edit functions all work accordingly and some minor responsive implementation for when the screen size gets smaller